sonaranalyzer-dotnet/SonarAnalyzer.sln

```diff
@@ -1,82 +1,82 @@
-﻿
```

Comment:
It seems like there is no real change so I would undo all changes on this file. This is confusing.